README.md

```diff
@@ -1,6 +1,28 @@
+# Pursuit-Core-Web-React-Lifecycle-Methods-Lab
+
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+## Tasks
+
+Build an app that has the following functionality:
+
+- Users can create a new Todo by typing text into an `input` inside a `form` and pressing enter
+- Shows a list of todos with id `#todos` underneath that input using instances of a `Todo` component
```

Comment:
`that input using instances` ... I'm not sure I follow what that means

Reply:
`that input` = the input from before, so rewording: `Shows (a list of todos with id `#todos`) (underneath the input) using instances of a `Todo` component`

Reply:
Thank you! It makes sense to me now and the previous phrasing also makes sense to me now. Reading comprehension fail from earlier.